css/style.css

```diff
@@ -666,3 +713,183 @@
   top: 19px;
   box-sizing: border-box;
 }
+
+/* ===========================
+Getting started
+===========================
+*/
+
+.get-started {
+  position: absolute;
+  width: 320px;
+  height: 813px;
+  left: -2px;
+  top: 4229px;
+  background: #f9f9fa;
+}
```

Comment:
- [ ] Kindly here change the `width: 320px` to `width: 100%` to fill the whole width, to match the figma design. 
- [ ] Kindly make the content of the section in the center to match the Figma design.

![image](https://user-images.githubusercontent.com/35535130/160608364-88d87fad-37e0-472b-bfb5-3b8768ff66fd.png)

Reply:
Hi @keroloussamy! Thanks for the quick reply 😄. I have:

- [ ] Attempted to put the width of the section to 100%, that spans over and above the 320px width container that's. In its stead, I've constricted all elements to be within the 320px and centered the form, 16px on either side of it.

- [ ] In regard to the bit that was to be worked out, constricted the inspector to 320px as demonstrated below. ♻️   
<img width="393" alt="Screenshot 2022-03-29 at 16 40 44" src="https://user-images.githubusercontent.com/40231363/160630707-2b90766b-68e2-48cc-9f66-aefec015b0c2.png">

- [ ] Hope these meet your review's expectations 😃.